loki/batch/scheduler.py

```diff
@@ -120,14 +120,16 @@ class Scheduler:
         By default a full parse is executed, use this flag to suppress.
     frontend : :any:`Frontend`, optional
         Frontend to use for full parse of source files (default :any:`FP`).
+    builddir : str or path
+        Directory for the output to be written to
     """
 
     # TODO: Should be user-definable!
     source_suffixes = ['.f90', '.F90', '.f', '.F']
 
     def __init__(self, paths, config=None, seed_routines=None, preprocess=False,
                  includes=None, defines=None, definitions=None, xmods=None,
-                 omni_includes=None, full_parse=True, frontend=FP):
+                 omni_includes=None, full_parse=True, frontend=FP, builddir=None):
```

Comment:
I would suggest we use a more generic term than `builddir` here, for example `output_path`. There is no reason that this needs to be an actual build directory, and for Loki processing that is not integrated with a build the name wouldn't make much sense.
It's just that in our use case we happen to specify the build directory as the output path

Reply:
Yes, good point! The `paths` argument suffer from the same misnomer but that's a separate issue